test/math_util_test.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "../src/math_util"
+require "test/unit"
+
+class TestMathUtil < Test::Unit::TestCase
+
+# Test sum_even_numbers
+
+  def test_sum_even_numbers_returns_nil
+    assert_nil(MathUtil.sum_even_numbers(nil))
+  end
+
+  def test_sum_even_numbers_empty_array_returns_0
+    assert_equal(0, MathUtil.sum_even_numbers([]))
+  end
+
+  def test_sum_even_numbers_returns_0
+    assert_equal(0, MathUtil.sum_even_numbers([0, 1, 3, 5]))
+  end
+
+  def test_sum_even_numbers_returns_12
+    assert_equal(12, MathUtil.sum_even_numbers([0, 2, 4, 6]))
+  end
+
+  def test_sum_even_numbers_returns_8
+    assert_equal(8, MathUtil.sum_even_numbers([0, -2, 4, 6]))
+  end
+
+# Test two_largest_numbers
+
+  def test_two_largest_numbers_returns_nil
+    assert_nil(MathUtil.two_largest_numbers(nil))
+  end
+
+  def test_two_largest_numbers_empty_array_returns_nil
+    assert_equal(nil, MathUtil.two_largest_numbers([]))
+  end
+
+  def test_two_largest_numbers_one_element_returns_nil
+    assert_equal(nil, MathUtil.two_largest_numbers([0]))
+  end
+
+  def test_two_largest_numbers_returns_4_and_3
```

Comment:
Usually when you describe a test, you should mention the function you are testing, the context that you are testing and how you expect it to behave:
This is good: https://github.com/Agilefreaks/Test/pull/36/files#diff-6cf82e7fbd372cdd4efcc55f35b88704R12
The line where this comment is attached to is not